src/styles/product-cart.scss

```diff
@@ -0,0 +1,138 @@
+@import "./variables.scss";
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  transition: transform 300ms ease;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+    transition: color 300ms ease;
+  }
+
+  &:hover &__name {
+    color: $card-name-hover-color;
+  }
+
+  &__product-code {
+    color: $secondary-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    margin-top: 4px;
+    align-self: flex-start;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 16px;
+  }
+
+  &__reviews-sum {
+    color: $main-color;
+    font-size: 10px;
+    height: 16px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price-elements {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-top: 24px;
+  }
+
+  &__price-text {
+    color: $secondary-color;
+    font-size: 12px;
+    font-weight: 400;
+    line-height: 18px;
+  }
+
+  &__price {
+    color: $main-color;
+    font-size: 16px;
+    font-weight: 700;
+    line-height: 18px;
+  }
```

Comment:
nesting

Reply:
I can`t code with nesting this line because when I coded product card I didn`t think about it and had some mistakes in the names

Reply:
If there are errors in the names, it should be corrected and nesting should be used to improve the readability of the code.